### ISSUE TITLE
Style poll results

### ISSUE THREAD

**Issue by asifkabani** [salalo/react-leaf-polls#35]
I was wondering what tweaks can I make to style my poll results like this:
<img width="552" alt="image" src="https://user-images.githubusercontent.com/9083341/180509530-78e22410-65b4-4cd5-94b1-a336e927eeb9.png">

**Comment by salalo** [comment]
Hi @asifkabani 

Currently there is no support to style individual bars of a poll. You probably could achieve something like this by implementing useRef hook on the fork yourself.

**Comment by asifkabani** [comment]
@salalo thanks for the information. Is there a way to have number of votes show up for an answer vs the percentage?

**Comment by salalo** [comment]
@asifkabani No, currently there is not but it would be fairly easy to implement, so maybe soon.

**Comment by asifkabani** [comment]
@salalo one other thing, if I set the `isVoted` prop to true, wondering why the percentage values don't show up

Here is my code:

```
const PollResults = ({ question, handleVote }) => {
  const themeData = {
    textColor: "#2B355C",
    mainColor: "#f3e7fa",
    backgroundColor: "#F5F5F7",
    alignment: "end",
  };

  const resData = [
    { id: 0, text: "React", votes: 9 },
    { id: 1, text: "Vue", votes: 7 },
    { id: 2, text: "Angular", votes: 2 },
  ];

  return (
    <>
      <Text fontWeight="600" fontSize="12px" color="#2B355C" mb="1">
        {question}
      </Text>
      <LeafPoll
        type="multiple"
        results={resData}
        theme={themeData}
        onVote={handleVote}
        isVoted={true}
        isVotedId={1}
      />
    </>
  );
};

export default PollResults;
```

![image](https://user-images.githubusercontent.com/9083341/180857928-5aa05bae-fe9b-4ac1-95c0-73d318834f7c.png)

**Comment by salalo** [comment]
@asifkabani I've seen that bug but it's pretty irregular, I'll try to replicate and fix it. You can try different version of the package to see if it helps.

**Comment by asifkabani** [comment]
@salalo I want to see if I can help contribute to this package. Is there a guide I can follow to figure out:

- After I fork, how do I use the forked version
- How do I see the source code to modify and check if it works in my local environment
- submit a PR for the fix

**Comment by hwb15** [comment]
> @asifkabani I've seen that bug but it's pretty irregular, I'll try to replicate and fix it. You can try different version of the package to see if it helps.

I'll take a look at this also and try replicate, think I've seen a similar issue where I use it in prod
